autoload/elm.vim

```diff
@@ -62,6 +62,9 @@ fun! elm#Format()
 		let old_fileformat = &fileformat
 		call rename(l:tmpname, expand('%'))
 		silent edit!
+    if g:elm_format_two_spaces == 1
```

Comment:
I love this hack!

I actually added a `--tabsize` option to elm-format ( https://github.com/specious/elm-format/commit/3d6aaff12dc3c2eedb3f246a9393f6f24e969216 ) in part as a contribution to this quintessential discussion - https://github.com/avh4/elm-format/issues/210

Reply:
That would be much preferable to using sed.